metric-anomaly-data-model/src/main/proto/org/hypertrace/alert/engine/eventcondition/config/service/v1/metric_anomaly_event_condition.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.alert.engine.eventcondition.config.service.v1;
+
+message MetricAnomalyEventCondition {
+  MetricSelection metric_selection = 1;
+  repeated ViolationCondition violation_condition = 2;
+}
+
+message MetricSelection {
+  Attribute metric_attribute = 1;
```

Comment:
```suggestion
  Attribute metric = 1;
```

Reply:
I had double thought on naming this? Do you guys prefer between `metric` vs `metric_attribute`? I am inclined towards just naming it as `metric`.

Reply:
I'd be fine with either `attribute` or `metric_attribute`. A metric in our system (at least today), is a combination of an aggregation and an attribute, so on its own, this value doesn't represent a metric.

Reply:
+1 for `metric_attribute`

Reply:
Sounds good.